tests/unit/plugins/module_utils/test_system_access_users_utils.py

```diff
@@ -518,52 +482,14 @@ def test_user_from_ansible_module_params_with_not_existing_group(
         "uid": "1000",
         "groups": ["not_existing_group"],
     }
-
     with UserSet(sample_config_path) as user_set:
         with pytest.raises(OPNSenseGroupNotFoundError) as excinfo:
```

Comment:
```suggestion
        with pytest.raises(OPNsenseGroupNotFoundError) as excinfo:
```

Reply:
Thanks for the suggestion, fixed in 2ee63a33df46b4308a4444edad3c7ed3b9172945.